mobile.css

```diff
@@ -1410,3 +1410,272 @@ button:hover {
   line-height: 24px;
   color: #fff;
 }
+
+.email-icon {
+  display: none;
+}
+
+.item-links {
+  display: none;
+}
+
+.big-circle1 {
+  display: none;
+}
+
+.big-circle2 {
+  display: none;
+}
+
+.desk1 {
+  display: none;
+}
+
+.desk2 {
+  display: none;
+}
+
+.work-btn2 {
+  display: none;
+}
+
+.desk3 {
+  display: none;
+}
+
+.desk4 {
+  display: none;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 24px;
+  position: absolute;
+  width: 340px;
+  height: 824px;
+  left: 0;
+  top: 4516px;
+}
+
+.contact-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  position: absolute;
+  width: 327px;
+  height: 164px;
+  left: 12px;
+  top: 120px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  position: absolute;
+  width: 327px;
+  height: 392px;
+  left: 12px;
+  top: 324px;
+  justify-content: center;
+}
+
+.contact-me h2 {
+  position: static;
+  width: 327px;
+  height: 35px;
+  left: 0;
+  top: 0;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  text-transform: uppercase;
+  justify-content: center;
+  color: #9390ff;
+}
+
+.contact-me h2:last-of-type {
+  display: none;
+}
+
+.contact-me p {
+  position: absolute;
+  width: 327px;
+  height: 105px;
+  left: 0;
+  top: 59px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #f5f5ff;
+  justify-content: center;
+}
+
+#input {
+  position: absolute;
+  height: 38px;
+  left: 0;
+  right: 0;
+  top: 0;
+  outline: 1px solid #fff;
+  width: 100%;
+}
+
+#email {
+  position: absolute;
+  left: 0;
+  right: 0;
+  height: 38px;
+  top: 17.66%;
+  outline: 1px solid #fff;
+  width: 100%;
+}
+
+#message {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 35.32%;
+  outline: 1px solid #fff;
+  max-width: 100%;
+  width: 100%;
+  max-height: 70%;
+}
+
+.submit-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  position: absolute;
+  width: 89px;
+  height: 28px;
+  left: 118px;
+  bottom: 1%;
+  background-color: #7d73fc;
+  color: #fff;
+  text-transform: capitalize;
+  font-family: 'Roboto', sans-serif;
+  justify-content: center;
+}
+
```

Comment:
- [ ] **REQUIRED:** It will be nice to have interactions for the button such as hover, active, disable and enable.